### PR TITLE
Actions_Panel

### PR DESCRIPTION
**Pull request by ShadowBrother**
-Added Actions_Panel to show actions's left per day and the current month/day

### CODE REVIEW

File: Assets/Scripts/MenuSystem/Actions_Panel.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+public class Actions_Panel : MonoBehaviour {
```
Comment (jmccormack200) [inline Assets/Scripts/MenuSystem/Actions_Panel.cs]: probably want to call this ActionsPanel not Actions_Panel

File: Assets/Scripts/SeedRandom.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@ public class SeedRandom : MonoBehaviour {
 
 	// Use this for initialization
 	void Start () {
-        Random.seed = 1337;
+        UnityEngine.Random.InitState(1337 * (int)Time.time);
```
Comment (jmccormack200) [inline Assets/Scripts/SeedRandom.cs:8]: damn nerds

**Review by jmccormack200 — APPROVED** [review]
:shipit:

File: Assets/Scripts/MenuSystem/Actions_Panel.cs (hunk 1 above)
Comment (ShadowBrother) [reply Assets/Scripts/MenuSystem/Actions_Panel.cs]: why do you hate underscores so much??

**Pull request merged**